src/App.js

```diff
@@ -0,0 +1,47 @@
+import { SideBar } from "./components/SideBar/index.js";
+import { DocumentPage } from "./components/DocumentEditor/index.js";
+import { initRouter } from "./router.js";
+import checkNewComponent from "./utils/checkNewComponent.js";
+
+export default function App({ $target }) {
+    const self = this;
+    checkNewComponent(App, self);
+
+    const sideBar = new SideBar({
+        $target,
+    });
+
+    const documentPage = new DocumentPage({
+        $target,
+        initialState: {
+            title: "",
+            content: "",
+            documentId: null,
+        },
+        updateSideBar: () => sideBar.render(),
+    });
+
+    const ToggleDocumentVisibility = (parent) => {
+        const childDocument = document.querySelector(".documentPage") || null;
```

Comment:
```suggestion
        const childDocument = document.querySelector(".documentPage");
```
만약 `document.querySelector`는 매칭된 엘리먼트가 존재하지 않을 경우 `null`을 반환하기 때문에 `T || null`과 같은 표현은 중복 표현입니다!

그리고 만약 좌변의 값이 null이나 undefined와 같은 값일 경우 우변의 값을 할당하고 싶을 때는 [Nullish Coalescing Operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing)를 사용해보시는 것도 좋을 것 같아요 ㅎㅎ 다만 지원 범위 꼭 잘 확인해주시구요!

Reply:
아 찾아보니까 querySelector는 매칭된 엘리먼트가 존재하지 않으면 멘토님 말씀대로 null을 반환해주네요!!😊
널 병합 연산자도 잘 알아보고 유용하게 쓰겠습니다!! 감사합니다 ㅎㅎㅎ 👍👍